src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/IEFormidlingClient.cs

```diff
@@ -17,14 +19,14 @@ public interface IEFormidlingClient
         /// </summary>
         /// <param name="subscription"> Object that is used to create subscription.</param>
         /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
-        Task<bool> SubscribeeFormidling(CreateSubscription subscription);
+        Task<bool> SubscribeeFormidling(CreateSubscription subscription, Dictionary<string, string> requestHeaders);
```

Comment:
Added a dictionary here to ensure that we don't have duplicates for header names. Could be a list as well if anyone has good arguments as to why. Trying to avoid to specify accessToken and authorizationToken specifically to make the client as generic as possible